### ISSUE TITLE
Support Accessibility regarding Announcing that the Recording has started

### ISSUE THREAD

**Issue by ansarmie** [SeleniumHQ/selenium-ide#964]
## 🐛 Bug Report

Fulfill the **W3 Standard** **1.3.1 - Info and Relationships** (Recommendation)

To support the use of the application by people with Hard of sight and Keyboard or Screen Reader users. And maximize compatibility with current and future user agents, including assistive technologies. Pull request #974

More info here:

- [Guideline 1.3 Adaptable](https://www.w3.org/TR/WCAG21/#adaptable)
- [Understanding Success Criterion 1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
- [Guideline 1.4 Distinguishable](https://www.w3.org/TR/WCAG21/#distinguishable)

## To Reproduce

- When the recording starts, there is a visual identification saying that the recording has started. A Screen Reader is not able to recognize that and read it. There is a possibility of using aria alert for that.

## Expected behavior

- The Screen Reader should be able to read back when the Recording has started.

## Project file reproducing this issue (highly encouraged)

This is a issue with the UI, not of a specific Selenium scenario.

## Environment

OS: Windows 10
Selenium IDE Version:  3.16.1
Browser: Google Chrome
Browser Version: 80.0.3987.116 (Official Build) (64-bit)

**Comment by tourdedave** [comment]
Fixed in [v3.17.0](https://github.com/SeleniumHQ/selenium-ide/releases/tag/v3.17.0)
